### ISSUE TITLE
Feature request: support for git svn dcommit

### ISSUE THREAD

**Issue by soltysek** [akonwi/git-plus#416]
I'm working with many git svn repos (as my company mostly uses svn..) and it would be nice to run dcommit. 

Now I have to edit files, commit changes, etc. But when I want to 'push' changes to svn repository I have to open console and run `git svn dcommit` manually.

Is there any chance to add this feature?

**Comment by akonwi** [comment]
This is a good point. I've been thinking about this. I'd like to have the push command do `dcommit` if the repo is using the git-svn bridge. That's something to come hopefully.

For now, you can use the `Run` command and enter `svn dcommit` from within atom. It may save you a few keystrokes

**Comment by soltysek** [comment]
Also it would be good if `pull` will trigger `svn rebase` for git-svn repos.

**Comment by akonwi** [comment]
I've just published a new version with a feature that allows users to define their own commonly used commands locally. Since this is a unique feature that may not be common to a lot of the user base of the package, I think it is a good candidate to test out as a custom command. Currently, custom commands are an experimental feature and I would love if you gave it a try and provided feedback. 

You can read about it here

**Comment by soltysek** [comment]
Could you provide some documentation/howto about defining this custom commands? For some reason:
```
atom.packages.onDidActivateInitialPackages (p) ->
if gitPlus = atom.packages.getActivePackage('git-plus')?.mainModule.provideService()
  gitPlus.registerCommand 'atom-text-editor', 'akonwi:rebase-from-svn', ->
    gitPlus.getRepo() # If there are multiple repos in the project, you will be prompted to select which to use
      .then (repo) -> gitPlus.run repo, 'svn rebase'

atom.packages.onDidActivateInitialPackages (p) ->
if gitPlus = atom.packages.getActivePackage('git-plus')?.mainModule.provideService()
  gitPlus.registerCommand 'atom-text-editor', 'akonwi:dcommit', ->
    gitPlus.getRepo() # If there are multiple repos in the project, you will be prompted to select which to use
      .then (repo) -> gitPlus.run repo, 'svn dcommit'
```
does not work for me

**Comment by akonwi** [comment]
I'm sorry, I just realized I didn't paste the hyperlink to the README in that last comment but it seems like you found it. 

What you have looks fine. Are you sure, you've enabled custom commands in the package settings?

**Comment by soltysek** [comment]
Yes, I have checkbox next to 'Custom Commands' in settings checked.

**Comment by soltysek** [comment]
After upgrading to version 7.0.5 it started to work! thanks :)
